client/java/generator/src/main/java/io/openlineage/client/SchemaParser.java

```diff
@@ -220,6 +229,11 @@ public String getDescription() {
       return description;
     }
 
+    @Override
+    public String toString() {
+      return "Field{name: " + name + ", type: " + type + "}";
```

Comment:
_Minor:_ We may want to standardize on `toString()` method overrides with lombok's [`ToString`](https://projectlombok.org/features/ToString) annotation. Looks like throughout the generator, a few formats are used.

Reply:
I fixed the inconsistencies thanks for pointing it out.
lombok is not a dependency at the moment so holding off on that.